bad.swift

```diff
@@ -0,0 +1,9 @@
+Group {
+  $0.command("login") { (name:String) in
```

Comment:
Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.